python-backend/app/api/parties/party/resources/party_resource.py

```diff
@@ -60,7 +61,11 @@ class PartyResource(Resource, UserMixin, ErrorMixin):
     @requires_role_mine_view
     def get(self, party_guid=None):
         if party_guid:
-            party = Party.find_by_party_guid(party_guid)
+            try:
+                party = Party.find_by_party_guid(party_guid)
+            except DBAPIError:
```

Comment:
Is this a common issue, there are alot of find_by_guid method?

Reply:
This was a 500 errror I found through postman that wasn't really part of the ticket.  I put a fix in after chatting with Nathan

Reply:
The find by guid is mostly used for the add contact drop down.